### ISSUE TITLE
Accessing a level that is not available gives an ugly error

### ISSUE THREAD

**Issue by Felienne** [Felienne/hedy#65]
For example: http://www.hedycode.com/hedy/8?lang=nl

**Comment by moorlag** [comment]
Explain error code: 500 (isn't it a 404?); 

404 Hedy hands in the air? 
500 Hedy confused 
403 Hedy hidden/crossed arms 
Suggestions: 

- Start over at level 1
- Start an issue on GitHub
- Send Felienne a tweet
- Start [BJOC](www.bjoc.nl)
- Start [Python in de klas](www.pythonindeklas.nl)

**Comment by moorlag** [comment]
[pull 66](https://github.com/Felienne/hedy/pull/66) resolves part of this problem.

**Comment by Felienne** [comment]
Yeah would be cool to have a few Hedy error pages with images! This is not a 404 because the server tries to access a file that is not there btw. Will look at the PR later.

**Comment by Felienne** [comment]
This can be closed, specific issue is fixed. Better error pages will be a new issue

**Comment by Felienne** [comment]
Ah turns out I forgot the links changed, haha. So also the issue of the links still needs to be resolved.

**Comment by fpereiro** [comment]
Hi @Felienne ! Reviewing the issue, I don't understand what's the issue regarding the links that should be fixed. Perhaps I can help with this, since it seems to be quite web related :).

**Comment by Felienne** [comment]
@fpereiro As you can see I am cleaning stuff up haha!

This issue is still open:

![image](https://user-images.githubusercontent.com/1003685/133444546-cb1e922b-870e-488f-b333-fb62f97a9865.png)

Can we make this into a nice 404, or even better a redirect to something we do have?

**Comment by fpereiro** [comment]
Hi @Felienne ! I think we could add a 404 page to catch misdirected requests. In this particular case, we could also redirect them to the first level of Hedy if they select a level that doesn't exist.

**Comment by Felienne** [comment]
Yes, let's do two things:

* Redirect for his issue
* Making a 404 page for other unfound pages

What do you think? Split this issue in two or do both for this issue?

**Comment by fpereiro** [comment]
The redirect is so simple, that I'd do it all on this issue.

Could we have perhaps a Felienne Original for the 404? :D

**Comment by Felienne** [comment]
Tadaaa!  
![IMG_A00CB4ECA7B4-1](https://user-images.githubusercontent.com/1003685/134522681-73b4cb1e-f95e-42b0-a0ed-d8289c273d5d.jpeg)

**Comment by Felienne** [comment]
Tadaaa!
![IMG_3834B719AAC3-1](https://user-images.githubusercontent.com/1003685/134522960-fa11ec75-4413-4466-86bb-a295dbe08c22.jpeg)

**Comment by Felienne** [comment]
And a 500 drawing to
![IMG_E36B2043025B-1](https://user-images.githubusercontent.com/1003685/134524155-27141deb-1859-4035-b0b6-03991e25432a.jpeg)
o!
